### PR TITLE
Moved the search bar display into the fetch callback. We should only …

### PR DESCRIPTION
**Pull request by jdizoglio**
…display a search bar if we set the hidden property and we have folder items.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jdizoglio has signed the CLA. Thanks for the pull request!

File: BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXFolderViewController.m [hunk 1]
```diff
@@ -414,6 +413,12 @@ - (void)fetchItemsWithCompletion:(void (^)(NSArray *items, BOOL fromCache, NSErr
         [BOXDispatchHelper callCompletionBlock:^{
             if (items.count > 0 && !error) {
```
Comment (boxtcarpel) [inline BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXFolderViewController.m]: We want to handle the other case as well, of no results

Comment (jdizoglio) [inline BoxBrowseSDK/BoxBrowseSDK/Controllers/BOXFolderViewController.m]: done

**Comment by boxtcarpel** [comment]
+2 LGTM

:bouquet:

**Pull request merged**